src/convertors/designspace.rs

```diff
@@ -228,7 +216,7 @@ fn load_font_info(font: &mut Font, info: &norad::FontInfo) {
     }
 }
 
-fn load_instances(font: &mut Font, instances: &[DSInstance]) {
+fn load_instances(_font: &mut Font, _instances: &[DSInstance]) {
```

Comment:
Could we leave this warning, please? It isn’t OK that the variable is unused - it should be used because there should be code which uses it!